pkg/provider/azure_vmssflex.go

```diff
@@ -65,11 +66,11 @@ func newFlexScaleSet(az *Cloud) (VMSet, error) {
 	}
 
 	var err error
-	fs.vmssFlexCache, err = fs.newVmssFlexCache()
+	fs.vmssFlexCache, err = fs.newVmssFlexCache(context.Background())
```

Comment:
should add ctx parameter in `newFlexScaleSet`, the intention is that the outside consumer, .e.g. csi driver would pass the ctx in outside